contracts/core/libraries/Payouts.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+contract Payouts {
+    struct PayoutSummary {
+        address recipient;
+        uint256 amount;
+        uint256 percentage;
+    }
+}
```

Comment:
Should this be in something like `strategies/libraries/`? I'm not sure, just seems like it doesn't belong in `core/` since it's not used anywhere in there.

Reply:
I agree, thinking about where to put it 🤔 and others that may evolve like it.

Reply:
removing this for now. we'll handle this when we create libraries/module